simple/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
-name = "{{project-name}}"
+name = "templates-qol"
 version = "0.1.0"
-authors = ["{{authors}}"]
+authors = ["vm910"]
```

Comment:
Can you revert these changes along with the ones in `Cargo.lock` and `main.rs`? The strings that are surrounded by double curly braces (e.g. `{{project-name}}`) are template variables so we shouldn't change them 🐻